alfasim_sdk_api/detail/bootstrap_linux.h

```diff
@@ -10,6 +10,7 @@
 
 #define MAX_PATH 32767
 
+// See DOCs in bootstrap_win.h
```

Comment:
Actually, they are a bit different:

    Load ALFAsim-SDK API shared object (so file).

    @param[out] api ALFAsim-SDK API.
    @return An #sdk_load_error_code value.

Reply:
Humm, I think we should unify it, using a generic Docs.

If we don't have how to do that, I think it should be:
```
Load ALFAsim-SDK API shared object (Linux) or dll file (Windows).

@param[out] api ALFAsim-SDK API.
@return An #sdk_load_error_code value.
``` 
What do you think, @tarcisiofischer?